src/interfaces/correction-interfaces/correct-name-interface.ts

```diff
@@ -1,47 +1,8 @@
-import { NameRequestApplicantIF } from '@/interfaces/'
-import { CorrectNameOptions, NameRequestStates } from '@/enums/'
-import { NrRequestActionCodes } from '@bcrs-shared-components/enums'
-import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module/'
+import { CorrectNameOptions } from '@/enums/'
 
 export interface CorrectNameOptionIF {
-  id: CorrectNameOptions,
-  title: string,
-  description?: string,
+  id: CorrectNameOptions
+  title: string
+  description?: string
   component: any
 }
-
-export interface NrCorrectionIF {
```

Comment:
Either we don't care about these, or we now import them from the shared interfaces 🎉